Makefile

```diff
@@ -41,7 +41,7 @@ migrate: .env
 
 shell: .env .docker-build
 	# Use `-u 0` to automatically become root in the shell
-	docker-compose run --user 0 web bash
+	docker-compose run --user 0 --service-ports web bash
```

Comment:
A bunch of lines in the docs require `--service-ports`. Instead of having two ways to invoke the shell, I figure it's easier to invoke it all the time with service ports enabled.